ReactWindows/js/Components/Picker/PickerAndroid.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule PickerAndroid
+ */
+'use strict';
+
```

Comment:
See [https://github.com/facebook/react-native/blob/master/Libraries/Components/Picker/PickerAndroid.ios.js](https://github.com/facebook/react-native/blob/master/Libraries/Components/Picker/PickerAndroid.ios.js)

You'll want to rename this file PickerAndroid.windows.js